website/src/hooks/useFontDownload.tsx

```diff
@@ -1,22 +1,26 @@
 import { useEffect, useState } from "react";
 
 import { MetadataProps } from "../@types/[font]";
+import { fontsourceDownload } from "../utils/fontsourceUtils";
 
 const useFontDownload = (metadata: MetadataProps, downloadLink: string) => {
   const [fontLoaded, setFontLoaded] = useState(false);
 
   useEffect(() => {
     setFontLoaded(false);
 
-    // Fetch font file
-    new FontFace(metadata.fontName, `url(${downloadLink})`)
-      .load()
-      .then((result) => {
-        document.fonts.add(result);
-        setFontLoaded(true);
-      })
-      .catch((error) => console.log(error));
-  }, [metadata.fontId, metadata.fontName, downloadLink]);
+    fetch(downloadLink)
+      .then((res) => res.text())
+      .then((cssFile) => {
+        document.getElementById("font-preview-css").innerText =
+          cssFile.replaceAll(
```

Comment:
Could we get a comment explaining this function and regex more? For future reference.